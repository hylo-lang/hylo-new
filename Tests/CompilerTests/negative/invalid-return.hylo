//! no-std

struct A {}

fun f0() {
  A()
}

fun f1() {
  return A()
}

fun f2() -> A {
  return
}
