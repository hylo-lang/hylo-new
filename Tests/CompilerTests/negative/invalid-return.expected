negative/invalid-return.hylo:6.3-4: error: type 'A' has no static member 'init'
  A()
  ^
negative/invalid-return.hylo:10.10-11: error: type 'A' has no static member 'init'
  return A()
         ^
negative/invalid-return.hylo:14.3: error: expected value of type 'A'
  return
  ^
